Tests/UnitTests/Purchasing/Purchases/PurchasesTransactionHandlingTests.swift

```diff
@@ -102,27 +104,35 @@ class PurchasesTransactionHandlingTests: BasePurchasesTests {
     }
 
     func testDelegateIsCalledTwiceIfCustomerInfoTheDifferent() throws {
-        let customerInfo1 = try CustomerInfo(data: [
-            "request_date": "2019-08-16T10:30:42Z",
-            "subscriber": [
-                "first_seen": "2019-07-17T00:05:54Z",
-                "original_app_user_id": "app_user_id",
-                "subscriptions": [:] as [String: Any],
-                "other_purchases": [:] as [String: Any],
-                "original_application_version": "1.0"
-            ] as [String: Any]
-        ])
+        let delegateReceivedCount = self.purchasesDelegate.customerInfoReceivedCount
 
-        let customerInfo2 = try CustomerInfo(data: [
-            "request_date": "2019-08-16T10:30:42Z",
-            "subscriber": [
-                "first_seen": "2019-07-17T00:05:54Z",
-                "original_app_user_id": "app_user_id",
-                "subscriptions": [:] as [String: Any],
-                "other_purchases": [:] as [String: Any],
-                "original_application_version": "2.0"
-            ] as [String: Any]
-        ])
+        let payment = SKPayment(product: self.product)
+
+        let transaction1 = MockTransaction()
+        transaction1.mockPayment = payment
+        transaction1.mockState = .purchasing
+
+        let transaction2 = MockTransaction()
+        transaction2.mockPayment = payment
+        transaction2.mockState = .purchasing
+
+        try self.delegate.storeKit1Wrapper(self.storeKit1Wrapper, updatedTransaction: transaction1)
+        try self.delegate.storeKit1Wrapper(self.storeKit1Wrapper, updatedTransaction: transaction2)
```

Comment:
This test no longer had the same semantics because it used the same transaction, which was a shortcut. I made this reflect a more normal flow, and added another one below that uses the same transaction, for which the second one we don't post